cfme/tests/automate/test_namespace.py

```diff
@@ -20,11 +21,16 @@
     params=[ta.a_namespace, ta.a_namespace_with_path])
 def namespace(request):
     # don't test with existing paths on upstream (there aren't any)
-    if request.param is ta.a_namespace_with_path and version.current_version() >= "5.3":
+    if request.param is ta.a_namespace_with_path and version.current_version() == version.LATEST:
         pytest.skip("don't test with existing paths on upstream (there aren't any)")
     return request.param()
 
 
+@pytest.fixture
+def setup_single_provider():
```

Comment:
Seems like this should be in `fixtures.providers`, I keep seeing examples of it in PRs.